notification/tests.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from django.utils import timezone
+from notification.models import Notification, NotificationType
+
+
+@pytest.fixture
+def notification(player):
+    return Notification.objects.create(
+        player=player,
+        sent_time=timezone.now(),
+        message='Test notification',
+        notification_type=NotificationType.WEBSITE,
+        is_read=False
+    )
+
+
+@pytest.mark.django_db
+class TestNotification:
+    def test_create_notification(self, notification):
+        assert isinstance(notification, Notification)
+
+    def test_persisted_notification_in_database(self, notification):
+        assert notification in Notification.objects.all()
+
+    def test_delete_notification_from_database(self, notification):
+        notification.delete()
+        with pytest.raises(Notification.DoesNotExist):
+            Notification.objects.get(pk=notification.pk)
+
+    def test_mark_as_read_notification(self, notification):
+        Notification.mark_notification_as_read(notification)
```

Comment:
After making this function non-static, you can enjoy the benefits it offers
```suggestion
        notification.mark_notification_as_read()
```

Reply:
Thanks for this tip, 
Done